### PR TITLE
Convert site updates to React

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-project-management/issues/275.

### CODE REVIEW

**Comment by iHiD** [comment]
Sweet. This looks good!

File: app/javascript/components/common/site-updates-list/SiteUpdate.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                src={maker.avatarUrl}
+                handle={maker.handle}
+                className="face"
+              />
+            ))}
+          </div>
+        </div>
+        {update.expanded ? (
+          <ExpandedInfo
+            IconComponent={Icon}
```
Comment (iHiD) [inline app/javascript/components/common/site-updates-list/SiteUpdate.tsx]: This bit's not quite right. If you check back on the loom you'll see that this should always be the `update` context, never the `track` context in the expanded block.

**Comment by iHiD** [comment]
@kntsoriano Just checking you saw this one?

**Comment by kntsoriano** [comment]
@iHiD Should be fixed now :)

**Comment by iHiD** [comment]
Thanks! :)

**Pull request merged**